### DISCUSSION TITLE
A portable Q8 version for Windows?

### DISCUSSION THREAD

**Discussion by kuba-orlik** [ImageMagick/ImageMagick#2093] [Help] [answered]
I'm in need of a portable version of IM with Q8 for Windows. The [downloads page](https://imagemagick.org/script/download.php) lists only Q16 versions for the portable variant. 

I've tried downloading the Q8 installer and then using the exe copied from Program Files, but it didn't work (could find `colors.xml` file, even though I copied it over with the exe.

My next attempt was to compile the portable Q8 version myself, but after 3 days of even trying to have VS running on my machine, I gave up. 

I'd be very thankful if someone posted a compiled Q8 portable version. 

On another note, is there a specific reason why there are no Q8 portable builds on the download page?

**Comment by dlemstra** [comment]
We don't have a Q8 build on the downloads page because we will need to configure this. At some point we will move from AppVeyor to GitHub actions. That will be the moment that we can take a look at this again and add a Q8 and Q16-HDRI portable version.

**Comment by dlemstra** [comment, marked as answer]
We published a new release yesterday that uses GitHub actions and we have added a portable version of the Q8 build.
